ddl.dic

```diff
@@ -993,6 +993,28 @@ save_enumeration.def_index_id
 
 save_
 
+save_enumeration.def_index_ids
+
+    _definition.id                '_enumeration.def_index_ids'
+    _definition.class             Attribute
+    _definition.update            2023-06-10
+    _description.text
+;
+    Specifies the data names of items that are collectively used to identify
+    the suitable default value in the ENUMERATION_DEFAULT category loop.
+    The values of these items are used to construct an ordered list which is
+    checked against the values of the _enumeration_defaults.index attribute.
+;
+    _name.category_id             enumeration
+    _name.object_id               def_index_ids
+    _type.purpose                 Identify
+    _type.source                  Assigned
+    _type.container               Array
```

Comment:
```suggestion
    _type.container               List
```

Reply:
I guess having an `Array` there is not incorrect, but I guess it was originally targeted more towards numeric types, mathematical operations, etc.

Reply:
I'm just going from the contents definitions.

Reply:
Agree with `List`.

Reply:
Ok. Changed the container type to `List` and updated the category name in the description.